broker.yaml

```diff
@@ -0,0 +1,717 @@
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                      type: string
+                      description: subscriptionId
+                      example: subscriptionId
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+        default:
+          $ref: '#/components/responses/Error'
+    delete:
+      summary: 'Delete service registry'
+      operationId: d
+      tags:
+        - services-from-registry
+      parameters:
+        - in: cookie
+          name: userAccessToken
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: 'The service registry is removed successfully'
+          content:
+            application/json:
+              schema:
+                type: string
+                example: "Deleted successfully"
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+        default:
+          $ref: '#/components/responses/Error'  
+components:
+  securitySchemes:
+    bearerAuth:
+      type: http
+      scheme: bearer
+      bearerFormat: JWT
+  responses:
+    Unauthorized:
+      description: 'Unauthorized ([RFC 7235](https://tools.ietf.org/html/rfc7235))'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Error'
+    Forbidden:
+      description: 'Forbidden ([RFC 7231](https://tools.ietf.org/html/rfc7231))'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Error'
+    NotFound:
+      description: 'Not found ([RFC 7231](https://tools.ietf.org/html/rfc7231))'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Error'
+    InternalServerError:
+      description: 'Internal server error ([RFC 7231](https://tools.ietf.org/html/rfc7231))'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Error'
+    Error:
+      description: 'Unexpected error ([RFC 7231](https://tools.ietf.org/html/rfc7231))'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Error'
+  
+  schemas:
+    Service:
+      type: object
+      properties:
+        projectId:
+            type: string
+            description: Unique id generated for each project
+            example: project_12
+        callbackUrl:
+            type: string
+            description: URL on which CD is listening for image updates.
+            example: https://ec2-54-203-145-132.compute-1.amazonaws.com/update
+        accessToken:
```

Comment:
In which occasions we want to show the `accessToken`? 

The broker creates the token and returns it to "operations" in the POST. It makes sense. But then, do we want to show it as well in the GET and PUT? It is fine for me, but I think it is something to consider.

Reply:
Yes, exactly we need not show `accessToken` again. I'll update it. Thanks!